components/HomeScreen.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from "react";
+import { StyleSheet, View, FlatList } from "react-native";
+import { useSelector } from "react-redux";
+import ArticleServices from "../modules/ArticleServices";
+import { StatusBar } from 'expo-status-bar'
+import DisplayArticlesList from './DisplayArticlesList'
+
+const styles = StyleSheet.create({
```

Comment:
For consistency try to have our `const styles` in the end (or top) on the component.

Reply:
done! :)